src/support/timestamp.c

```diff
@@ -251,9 +251,9 @@ __wt_time_aggregate_validate(
           "aggregate %s",
           __wt_time_aggregate_to_string(ta, time_string[0]));
 
-    if (ta->oldest_start_txn > ta->newest_stop_txn)
+    if (ta->newest_txn > ta->newest_stop_txn)
```

Comment:
Couldn't this be true if we had a newer start txn greater than a stop txn?

Reply:
No. If we have a key without a tombstone, the `newest_stop_txn` will be WT_TXN_MAX. Even if all the keys are having a tombstone, still the tombstone transaction will be the latest transaction.